[docs]  Update Orbit guide's structure, add Windows installation instructions (#28135)
expo/expo: docs/pages/build/orbit.mdx

---
title: Expo Orbit
description: Accelerate your development workflow with one-click build and update launches and simulator management.
---

import { Terminal } from '~/ui/components/Snippet';
import Video from '~/components/plugins/Video';
import { Tabs, Tab } from '~/ui/components/Tabs';

> Expo Orbit for macOS and Windows makes it faster to install and run builds from EAS, local files, or run Snack projects, on simulators and physical devices.

Author: ```suggestion
Expo Orbit for macOS and Windows enables faster to install and launch builds or updates from EAS, local files, or run Snack projects, on simulators and physical devices.
```

Let's also update the intro to add the context about "updates".

<Video file="orbit/basic-features.mp4" />

## Why Orbit

> Before Orbit, installing builds from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.

Author: ```suggestion
Before Orbit, installing builds or updates from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
```

> Before Orbit, installing builds or updates from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.

Author: ```suggestion
Before Orbit, installing builds or updates from EAS (on Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible.
```

Fix typo when addressing devices part.
